packages/video_player/video_player/CHANGELOG.md

```diff
@@ -19,6 +23,7 @@
 ## 0.10.12+3
 
 * Avoiding uses or overrides a deprecated API in `VideoPlayerPlugin` class.
+>>>>>>> master
```

Comment:
Leftovers of a merge here

Reply:
Fixed in the child PR #3 : https://github.com/flutter/plugins/pull/3138/files#diff-82e451e154983ecbae7c1dfb22c5dc4324ecd25cc2f7ab9a933e2abf4d14fe4a

Reply:
Can I close this PR then in favour of the other 3; #3136 #3137 and #3138?

Reply:
@Salakar If that's best then yes, whatever makes the most sense to avoid confusion. 

As a start I updated this PR to have 0 diffs so its obvious the changes are happening elsewhere.